app/src/full/java/io/homeassistant/companion/android/sensors/LocationSensorManager.kt

```diff
@@ -614,7 +623,7 @@ class LocationSensorManager : BroadcastReceiver(), SensorManager {
     }
 
     override val enabledByDefault: Boolean
-        get() = true
+        get() = false
```

Comment:
What was the reason for this change again? I think we kept this as `true` to ensure onboarding works.  Although onboarding has changed since the introduction of `hasSensor`.

Anyways if we need it to be `false` then we can remove the method as it defaults to `false`

Reply:
Good point. I think i need to tweak that one.

Right now, the location tracking switch defaults to "Location enabled" in the android settings.
https://github.com/home-assistant/android/blob/f38ec600e3cba15ebadec3e474f73410c78203c8/app/src/main/java/io/homeassistant/companion/android/onboarding/integration/MobileAppIntegrationFragment.kt#L75

If the location tracking switch is disabled and the user doesn't change the switch. location tracking will be disabled, but the sensor will be left enabled.
If the location tracking switch is enabled and the user doesn't change the switch, location tracking will be enabled.

With this change, i only switched this logic :dizzy_face:

We need to make sure, that the sensors will set accordingly to the default of location tracking option

Reply:
just remember a few things about this.  the `device_tracker` entity will always show up, the sensors themselves arent really sensors just house the logic for updating.  if location permission is not granted there won't be any location updates and the sensors should be disabled by default then.  I think?

Reply:
Yes, i would leave them disabled by default, if there is no additional reason behind this.

if i call `setLocationTracking(it.isChecked)` after
https://github.com/home-assistant/android/blob/f38ec600e3cba15ebadec3e474f73410c78203c8/app/src/main/java/io/homeassistant/companion/android/onboarding/integration/MobileAppIntegrationFragment.kt#L75

Then this should be ok, i think.
Location tracking then should be disabled/enabled accordingly to the android location setting.